### ISSUE TITLE
Razor Pages _ViewImports documentation could be more clear

### ISSUE THREAD

**Issue by blowdart** [aspnet/Mvc#7932]
_From @tstivers1990 on June 18, 2018 12:48_

https://docs.microsoft.com/en-us/aspnet/core/mvc/razor-pages/?view=aspnetcore-2.1&tabs=visual-studio#using-layouts-partials-templates-and-tag-helpers-with-razor-pages

The part of this documentation that details the `@namespace` declaration could be made more clear. It has led some people to believe `@namespace` should be used as though it is a `using` statement. See the linked StackOverflow answer below for an example of this. I believe part of the reason it isn't clear, is because it doesn't explain `@using` directives in the context of a `_ViewImports` file.

In short, the wording could be made more clear, and a section should be added detailing `@using` directives in the context of a `_ViewImports` file.

https://stackoverflow.com/a/50322359

_Copied from original issue: aspnet/Identity#1839_

**Comment by blowdart** [comment]
This doesn't appear to be an identity issue, so moving.

**Comment by tstivers1990** [comment]
Thanks. I derped and put it in the wrong place.

**Comment by mkArtakMSFT** [comment]
Thanks for bringing this up, @tstivers1990.
We'll try to clarify this.

**Comment by NTaylorMullen** [comment]
After reading through the article I think it is pretty concise in regards to what `@namespace` accomplishes. It's specific in what pieces it describes and only explains pieces that are essential to what's being asked of the user (they're adding an `@namespace`). Therefore, I don't think it's the right place to add information about the using. That's accomplished [elsewhere](https://docs.microsoft.com/en-us/aspnet/core/mvc/views/razor?view=aspnetcore-2.1#section-1).

All that being said i did find the last sentence in that paragraph a bit confusing, I'll remove it.

**Comment by NTaylorMullen** [comment]
1521b50a66fc1056bf186f2b9d81a4ba07783d5d
